packages/snowpack/src/commands/install.ts

```diff
@@ -14,22 +14,25 @@ import path from 'path';
 import rimraf from 'rimraf';
 import {InputOptions, OutputOptions, rollup, RollupError} from 'rollup';
 import validatePackageName from 'validate-npm-package-name';
-import {EnvVarReplacements, SnowpackConfig, SnowpackSourceFile} from '../config.js';
 import {resolveTargetsFromRemoteCDN} from '../resolve-remote.js';
 import {rollupPluginCatchUnresolved} from '../rollup-plugins/rollup-plugin-catch-unresolved.js';
 import {rollupPluginCatchFetch} from '../rollup-plugins/rollup-plugin-catch-fetch';
 import {rollupPluginCss} from '../rollup-plugins/rollup-plugin-css';
 import {rollupPluginDependencyCache} from '../rollup-plugins/rollup-plugin-remote-cdn.js';
-import {
-  DependencyStatsOutput,
-  rollupPluginDependencyStats,
-} from '../rollup-plugins/rollup-plugin-stats.js';
+import {rollupPluginDependencyStats} from '../rollup-plugins/rollup-plugin-stats.js';
 import {rollupPluginWrapInstallTargets} from '../rollup-plugins/rollup-plugin-wrap-install-targets';
-import {InstallTarget, scanDepList, scanImports, scanImportsFromFiles} from '../scan-imports.js';
```

Comment:
This is one example where we’re importing types from 3 different files, one of which we only were importing _for_ its types. In many cases, importing types could be a hunt for where they were defined.